paper-header-panel.html

```diff
@@ -405,16 +303,16 @@
         var shadowMode = MODE_CONFIGS.shadowMode[mode];
 
         if (this.shadow) {
-          this.toggleClass('has-shadow', true, this.header);
+          this.toggleClass('has-shadow', true, this.$.dropShadow);
 
         } else if (shadowMode === SHADOW_ALWAYS) {
-          this.toggleClass('has-shadow', true, this.header);
+          this.toggleClass('has-shadow', true, this.$.dropShadow);
 
         } else if (shadowMode === SHADOW_WHEN_SCROLLING && !atTop) {
```

Comment:
This is not really relevant to this PR, but as a convention we try to avoid having constants that are just hanging in a closure scope. These should be moved onto the element constructor, or somewhere else where someone outside the closure can access them. This can be done in a different PR.